src/index.js

```diff
@@ -96,11 +97,34 @@ export default function(input) {
         return [];
       }
 
-      const { prefixNode } = checkPrefix(trie, strPrefix);
+      let prefixNode;
```

Comment:
How about making prefixNode a ternary. I try to avoid using let wherever I can. Thoughts?